frontend/src/domain/documentationUnitHistoryLog.ts

```diff
@@ -0,0 +1,11 @@
+export type DocumentationUnitHistoryLog = {
+  id: string
+  createdAt?: string
+  documentationUnitId?: string
+  documentationOffice?: string
+  userId?: string
+  userName?: string
+  systemName?: string
```

Comment:
```suggestion
  createdBy?: string
```
userId brauchen wir nicht, die name property heißt im Domain object `createdBy`